firebase/src/core/data/repositories/git-repository.ts

```diff
@@ -0,0 +1,31 @@
+import { ShellGateway } from "@data/gateways/shell-gateway";
+
+export class GitRepository {
```

Comment:
Can we add tests for these?

Reply:
Maybe we could add docs for the class?

Reply:
Added small docs. Let me know what do you think